zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -422,6 +422,8 @@ def get_cluster_services(
     endpoints = get_all(kube_client, kube_client.get_endpoints, namespace, span=current_span)
     # number of endpoints per service
     endpoints_map = {e.name: len(e.obj['subsets']) for e in endpoints if e.obj.get('subsets')}
+    config_endpoints_map = {e.name: e.obj['metadata'].get('annotations', {}) for e in endpoints
+                            if e.name.endswith('-config')}
```

Comment:
I guess it's also good to check spilo label, otherwise it can stumble upon something else with `-config` in the name:
```
  labels:
    application: spilo
```